### DISCUSSION TITLE
do you guys know how to get rid of this border line in taskbar10 style? (already uncheck all of ease of access settings)

### DISCUSSION THREAD

**Discussion by trefeon** [valinet/ExplorerPatcher#2324] [Q&A]
![276512282-4e4898f1-5707-4445-a330-cf5281e5fc5d](https://github.com/valinet/ExplorerPatcher/assets/95156714/3cdc3105-2347-4f4b-aec6-2c32b299f8c5)
![image](https://github.com/valinet/ExplorerPatcher/assets/95156714/09183c4d-c3de-406a-8526-bd5f2b958768)
![image](https://github.com/valinet/ExplorerPatcher/assets/95156714/ffea9886-06f1-42a1-aa46-222dcb9e117f)

**Comment by pyrates999** [comment]
answered here: https://github.com/valinet/ExplorerPatcher/issues/2323
